documentation/source/experiment_monitoring.md

```diff
@@ -0,0 +1,103 @@
+# Third-party experiment monitoring
+
+SuperGradients supports out-of-the-box Weights & Biases (wandb) and ClearML. 
+
+
+Simply update your train parameters to specify your preferred third-party logging tool, and the Trainer will 
+take care of the rest, handling all the necessary logic and integration.
+
+### Tensorboard
+**requirements**: None
+
+Tensorboard is natively integrated into the training and validation steps. You can find how to use it in [this section](TODO:add_link_to_our_tn_description).
+
+
+### Weights & Biases
+**requirements**:
+- Install `wandb`
```

Comment:
before this line:
register to W&B and make sure you have a user name and token to connect to the service

Reply:
I added something like that 5168bb65f56074a9aa1cac07d0a28b19a55c7107